### PR TITLE
Drop Python 2.7 support

### PR DESCRIPTION
**Pull request by syldlb**
## Review

This PR closes #

* [x] Tests
* [x] Docs/comments
  * [x] *IN CASE YOU'VE CHANGED THE `formidable.yml` file*: run ``tox -e swagger-statics`` to rebuild the swagger static files **and** commit the diff.
* [x] Migration(s)
* [x] `CHANGELOG.rst` Updated
* [ ] Delete your branch

### CODE REVIEW

**Comment by syldlb** [comment]
Done, I also removed some encodings left

File: demo/tests/test_exception_handler.py [hunk 1]
```diff
@@ -1,9 +1,3 @@
-# -*- coding: utf-8 -*-
-"""
```
Comment (brunobord) [inline demo/tests/test_exception_handler.py]: why did you remove the docstring here?

File: docs/source/deprecations.rst [hunk 2]
```diff
@@ -2,6 +2,17 @@
 Deprecation timeline
 ====================
 
+From 3.3.0 to 4.0.0
```
Comment (brunobord) [inline docs/source/deprecations.rst]: maybe you could add the fact that this version isn't released yet.

File: docs/source/deprecations.rst [hunk 3]
```diff
@@ -2,8 +2,8 @@
 Deprecation timeline
 ====================
 
-From 3.3.0 to 4.0.0
-==========================
+From 3.3.0 to 4.0.0 (unreleased)
+===================
```
Comment (brunobord) [inline docs/source/deprecations.rst]: docs won't build. the title underline should be as long as the title itself :/

Comment (syldlb) [reply docs/source/deprecations.rst]: fixed!
